.github/workflows/realm-generator.yml

```diff
@@ -0,0 +1,64 @@
+name: Realm Generator
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+  pull_request:
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+jobs:
+  Generator_CI:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest] # windows is not tested ue to bug in how error messages are rendered
+        channel: [stable, beta]
+
+    defaults:
+      run:
+        working-directory: ./generator/
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name : Setup Dart SDK
+        uses: dart-lang/setup-dart@main
+        with:
+          sdk: ${{ matrix.channel }}
+
+      - name: Install dependencies
+        run: dart pub get
+
+      - name: Run generator tests
+        run: dart test --reporter expanded --coverage ./coverage/
+
+      - name: Generate generator coverage report
+        run: |
+          dart run coverage:format_coverage \
+            --in coverage/ \
+            --out ./coverage/lcov.info \
+            --check-ignore \
+            --lcov \
+            --packages .packages \
+            --report-on lib
```

Comment:
Should this have `continue-on-error: true`?

Reply:
`continue-on-error: true`  here will break the build? The way I read it this will fail the build currently  if `Generate generator coverage report` fails.

Reply:
No `continue-on-error: true` means that the job will continue, even if this step fails.

Reply:
This is my understanding as well. This is why I asked. So should this step have the same configuration?

Reply:
Sorry, now I understand.. was confusing with the next step.. my mistake. No - this does not depend on an external service. I think it should just break the build, if it fails

Reply:
I am not sure we care so much for the coverage and if tests are green than this should be a nice to have. But I don't expect this to fail to often or at all, so lets leave it as it is.